### ISSUE TITLE
Addition of Resnet18 Model in Application

### ISSUE THREAD

**Issue by MrinalTyagi** [keras-team/keras#15494]
If you open a GitHub issue, here is our policy:

It must be a bug, a feature request, or a significant problem with the documentation (for small docs fixes please send a PR instead).
The form below must be filled out.

**Here's why we have that policy:**.

Keras developers respond to issues. We want to focus on work that benefits the whole community, e.g., fixing bugs and adding features. Support only helps individuals. GitHub also notifies thousands of people when issues are filed. We want them to see you communicating an interesting problem, rather than being redirected to Stack Overflow.

**System information**.

TensorFlow version (you are using): 2.6.0
Are you willing to contribute it (Yes/No): Yes

**Describe the feature and the current behavior/state**. Addition of a new pre-trained model: Resnet18 in the application section.

Describe the feature clearly here. Be sure to convey here why the requested feature is needed. Any brief description of the use-case would help.  

**Will this change the current API? How?** Yes. Addition of a new Resnet18 pre-trained model in the application section.

**Who will benefit from this feature?** Researchers as well as individuals who want to fine-tune or train Resnet18 in Tensorflow without switching to PyTorch

**[Contributing](https://github.com/keras-team/keras/blob/master/CONTRIBUTING.md)**

- Do you want to contribute a PR? (yes/no): Yes
- If yes, please read [this page](https://github.com/keras-team/keras/blob/master/CONTRIBUTING.md) for instructions
- Briefly describe your candidate solution(if contributing):
Addition of following code to the api:

`tf.keras.applications.ResNet18( include_top=True, weights='imagenet', input_tensor=None, input_shape=None, pooling=None, classes=1000, classifier_activation='softmax', **kwargs )`

Use Case: This would allow individuals to fine-tune the ResNet18 model pre-trained on the imagenet dataset to be fine-tuned on their custom dataset. As the implementation of ResNet18 is already available in PyTorch and not in Tensorflow, sometimes it becomes difficult for developers with knowledge of Tensorflow and no prior knowledge of Pytorch to implement the following. Hopefully, this will allow better transfer learning for developers and reduce time in writing the ResNet18 model from scratch, then training it on the imagenet dataset and then fine-tuning it on their custom dataset. Let me know if this resolves the doubt or I need to provide more explanation.
[Paper for ResNet18 introduction and architecture](https://arxiv.org/pdf/1512.03385.pdf)

**Comment by innat** [comment]
@MrinalTyagi 
It's already been asked, [here](https://github.com/keras-team/keras/issues/15269)

**Comment by innat** [comment]
@jvishnuvardhan 
Sorry, it's not clear to me the above comment. Is there any model application in `tf-addons`? I think it's [well used](https://github.com/keras-team/keras-applications/issues/151) and worthy to add under `tf.keras.applicaitons`.

**Comment by MrinalTyagi** [comment]
> Duplicate of ##15269
> 
> Is it possible for you and @innat to add this to `tf-addons`. Based on usage, this may be moved under `tf.keras.applications`. Thanks!

yes I can surely try that but would require some more information on how to submit the required pr for that.

**Comment by MrinalTyagi** [comment]
> Duplicate of #https://github.com/keras-team/keras/issues/15269
> 
> 
> 
> Is it possible for you and @innat to add this to `tf-addons`. Based on usage, this may be moved under `tf.keras.applications`. Thanks!

any update on this?

**Comment by MrinalTyagi** [comment]
any update guys?

**Comment by vgangal101** [comment]
i can implement breadbreads implementation into the codebase, within a week or so

**Comment by KaleabTessera** [comment]
+1

**Comment by gowthamkpr** [comment]
@MrinalTyagi This issue is a duplicate of the issue #15269 and there is an active PR for this issue [here](https://github.com/keras-team/keras/pull/16363). Please close the issue as it has already been tracked using #15269. Thanks!
